sites/platform/src/development/file-transfer.md

```diff
@@ -166,3 +174,70 @@ If you're using UTF-8 encoded files on macOS,
 add the `--iconv=utf-8-mac,utf-8` flag to your `rsync` call.
 
 For more options, consult the [rsync documentation](https://man7.org/linux/man-pages/man1/rsync.1.html).
+
+### sftp
+
+You can use `sftp` to copy files to and from a remote environment.
+
+{{% version/specific %}}
+{{% note %}}
+<!-- Platform.sh -->
+
+`sftp` is supported on the Grid, but the following limitations apply:
+
+- You can only create `sftp` accounts with an existing {{% vendor/name %}} user and an SSH key. 
+  Custom users and passwords aren't supported.
+- `sftp` access cannot be limited to a specific directory.
+  Instead, access is given to **the whole application directory** and its mounts.
+
+`sftp` is also supported on Dedicated projects with different limitations and requirements.
+For more information, see the [{{% names/dedicated-gen-2 %}}](/dedicated-gen-2/architecture/options.md#sftp)
+and [{{% names/dedicated-gen-3 %}}](/dedicated-gen-3/options.md#sftp) sections.
+{{% /note %}}
+
+<--->
+<!-- Upsun -->
+{{% note %}}
+{{% vendor/name %}} supports `sftp`, but the following limitations apply:
+
+- You can only create `sftp` accounts with an existing {{% vendor/name %}} user and an SSH key. 
+  Custom users and passwords aren't supported.
+- `sftp` access cannot be limited to a specific directory.
+  Instead, access is given to **the whole application directory** and its mounts.
+{{% /note %}}
+{{% /version/specific %}}
+
+#### Open an `sftp` connection
+
+Run the following command:
+
+```bash
+sftp "$(platform ssh --pipe)"
+```
+
+When prompted, select the project and environment you want to connect to.
+The `sftp` connection is open once the `sftp>` prompt is displayed in your terminal.
+
+#### Download a file
+
+Say you want to download a `diagram.png` file from the `web/uploads` directory 
+(relative to the [app root](../create-apps/app-reference.md#root-directory)).
+To do so, run the following command:
+
+```
+sftp> get /app/web/uploads/diagram.png
```

Comment:
you dont _have_ to use the whole path to the file for the `get` and `put` commands. you can use relative paths, or `cd` into the target directory before issuing the command. But the full path should always work.

Reply:
Yes, now that you pointed this out, I think it makes more sense to use relative paths in the examples so users for ease of use/so users know they aren't required to use the whole path + consistency with `rsync` and `scp` examples.